lib/evss/mhvcf/get_in_flight_forms_request_form.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+require 'active_model'
+require 'common/models/attribute_types/utc_time'
+
+module EVSS
+  module MHVCF
+    class GetInFlightFormsRequestForm
+      extend ActiveModel::Naming
+      include ActiveModel::Validations
+      include Virtus.model(nullify_blank: true)
```

Comment:
Should probably create a Common::FormObject model that has these extend and includes, and just have this form inherit from that.